Raptacon/Robot-2023
# Added GoToDist and TurnToAngle commands to work with autonomous

## Types of changes


## Associated Issue(s)
Making an autonomous that drives forward based on a distance and turns based on degree. DOES NOT HAVE PID.

## Steps to Test
Change the values in auto.py and measure to see if it follows the values accurately-ish as it doesn't have PID yet.

## Changelog
We need a way to move in autonomous

## Why?
This makes it so we can actually move in autonomous, which is probably helpful.
